components/nimbus/android/build.gradle

```diff
@@ -3,9 +3,6 @@
 // READ THIS CAREFULLY before editing this file.
 // =============================================
```

Comment:
You'll need to update `/.buildconfig-android.yml` to track the movement of this gradle file.